server.go

```diff
@@ -1,16 +1,26 @@
 package main
 
 import (
+	"log"
 	"net/http"
 
+	"github.com/imadu/e-commerce-api/db"
 	"github.com/labstack/echo"
+	"github.com/subosito/gotenv"
 )
 
 func main() {
-	e := echo.New()
-	e.GET("/", func(c echo.Context) error {
+	err := gotenv.Load()
+	if err != nil {
+		log.Fatalf("Could not load env file: %+v", err)
```

Comment:
You should return early, to avoid fatal errors, as this won't stop it from running the next line